internal/features/features.go

```diff
@@ -13,8 +13,13 @@ const (
 
 var operatorControllerFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	// Add new feature gate definitions
+	// if you're adding a feature gate for fellow developers to use in ongoing development, use PreAlpha
 	// Ex: SomeFeature: {...}
-	PreflightPermissions: {Default: false, PreRelease: featuregate.Alpha},
+	PreflightPermissions: {
+		Default:       false,
+		PreRelease:    featuregate.PreAlpha, // keep this PreAlpha until done with feature development
+		LockToDefault: false,
```

Comment:
Should not be true?

Reply:
If we set `LockToDefault` to `true`, @ankitathomas reports that you cannot access the feature-gated code for testing by then setting the emulated version. So LockToDefault true would render it untestable.

Reply:
If we do not set it true:
- users will be able to opt-in / use a flag that do nothing
- then, we came back to the same concerns: https://github.com/operator-framework/operator-controller/pull/1666#discussion_r1935172236

Reply:
I think this is resolved?